cpp/src/io/json/read_json.cu

```diff
@@ -270,6 +274,15 @@ table_with_metadata read_batch(host_span<std::unique_ptr<datasource>> sources,
   return device_parse_nested_json(buffer, reader_opts, stream, mr);
 }
 
+size_t get_batch_size_upper_bound()
```

Comment:
A free function in a source file needs to be either static or embedded in an anonymous namespace to avoid potential symbol conflict.

Reply:
Moved all free functions to an anonymous namespace, thanks!